### PR TITLE
Adds new clown mobs for admin use

### PR DESCRIPTION
**Pull request by DaxDupont**
About The Pull Request
This PR adds several clown subtypes, an alternate version of the honkrender that spawns clown hulks (a subtype added in this PR), a clown spawning beacon/nest, and two types of clown spawning grenades. These new entities aren't available to players by default, but admins can spawn them in if they desire to add an extra element of pranking. The dmi for the clown mobs contains a fair number of additional clown sprites not used for any current entity. I decided to include them for the convenience of other coders.

Why It's Good For The Game
While the jovial zeitgeist of the clown is expressed in many areas of the game, this new code allows admins to give it the spotlight, surprising players both new and old with a fresh and horrifying take on one of the game's most loved characters. This PR is also perfect for admin April fools events.

Changelog
🆑 Dax Dupont and Pugnacious-Gnome
add: several new clown types of varying strength
add: spawner beacon/nest to spawn clown mobs
add: alternate version of the honkrender that spawns 5 clown hulks
add: new spawner grenade called stuffed C.L.U.W.N.E. which spawns 5 mutant clowns
imageadd: sprites for new clown mobs, grenades, and spawner as well as some additional clown sprites for future mobs/admemery
/🆑

### CODE REVIEW

**Comment by DaxDupont** [comment]
Requesting @ExcessiveUseOfCobblestone to review difference between new and old
@optimumtact no more syndicate items

**Comment by improvedname** [comment]
im horrified by this

**Comment by DaxDupont** [comment]
good

**Comment by ExcessiveUseOfCobblestone** [comment]
die die die die die

**Comment by SouDescolado** [comment]
I have no basis on my accusations and wont hold them to court, but why do I feel pugnacious is goof?

**Comment by OverlyGeneric** [comment]
> I have no basis on my accusations and wont hold them to court, but why do I feel pugnacious is goof?

isn't goof. just some reddit man that made spookies. goof probably has an alt but it isn't pugnacious.

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 1]
```diff
@@ -44,3 +46,232 @@
...
+		if(L.reagents)
+			L.reagents.add_reagent(attack_reagent, rand(1,5))
+
+/mob/living/simple_animal/hostile/retaliate/clown/lube
+	name = "Living Lube"
+	desc = "A puddle of lube brought to life by the honkmother."
+	icon_state = "lube"
+	icon_living = "lube"
+	turns_per_move = 1
+	response_help = "dips a finger into"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: 1 step from erp

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 2]
```diff
@@ -44,3 +46,232 @@
...
+	maxHealth = 200
+	health = 100
+	speed = -5
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	attacktext = "ferociously mauls"
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/xeno/bodypartless, /obj/effect/particle_effect/foam, /obj/item/soap)
+	attack_reagent = "methamphetamine"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: yikes

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 3]
```diff
@@ -44,3 +46,232 @@
...
+	icon_state = "blob"
+	icon_living = "blob"
+	speak = list("hey, buddy", "HONK!!!", "H-h-h-H-HOOOOONK!!!!", "HONKHONKHONK!!!", "HEY, BUCKO, GET BACK HERE!!!", "HOOOOOOOONK!!!")
+	emote_see = list("jiggles", "wobbles")
+	health = 130
+	mob_size = MOB_SIZE_LARGE
+	speed = 20
+	attacktext = "bounces off of"
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/xeno/bodypartless, /obj/effect/particle_effect/foam, /obj/item/soap, /obj/effect/gibspawner/generic, /obj/effect/gibspawner/generic/animal, /obj/effect/gibspawner/human/bodypartless, /obj/effect/gibspawner/human)
+	attack_reagent = "skewium"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: yikes

**Comment by ExcessiveUseOfCobblestone** [comment]
these icons are scary I wish I never reviewed this pr

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm (hunk 2 above)
Comment (DaxDupont) [reply code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: any suggestions for different reagent?

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm (hunk 3 above)
Comment (DaxDupont) [reply code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: any suggestions for different reagent?

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm (hunk 1 above)
Comment (DaxDupont) [reply code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: feature

**Comment by DaxDupont** [comment]
reagents changed to less bad

**Comment by ExcessiveUseOfCobblestone** [comment]
if we keep adding reagent applications to mobs it'll need to be bumped up to simple_animals since `poison` already exists (and now clowns have something similar).

**Comment by PKPenguin321** [comment]
Really hope this goes through this time

**Comment by ExcessiveUseOfCobblestone** [comment]
i forgot why I didn't merge this

**Pull request merged**